packages/block-library/src/navigation/edit/navigation-menu-selector.js

```diff
@@ -24,7 +24,6 @@ function NavigationMenuSelector(
 		onSelectNavigationMenu,
 		onSelectClassicMenu,
 		onCreateNew,
-		showManageActions = false,
```

Comment:
Honestly this depends entirely on the new UX you've implemented. I believe it was originally there in order to allow us to determine whether or not the "Manage menus" options were displayed based on the context in which the NavigationMenuSelector was being shown.

If those different context have now been removed then this requirement is no longer valid and you can remove it.